README.md

```diff
@@ -83,3 +83,55 @@ You can run these tests without Docker by using this command 'poetry run pytest'
 A Github actions workflow has been added, this ignores any updates to this document (README.md)
 The workflow will test building the docker container on push and pull requests. The workflow will run the tests on the main branch and exercise-7#2 branch you can amend this in ci-workflow.yml for any branch you wish.
 Open Github and then open actions to view the workflow tests to see if they have run successfully.
+Images are now deployed to DockerHub for development you can access using this link: 'https://hub.docker.com/repository/docker/helencurtis02/todo-app-dev/general'
+
+You can now run your production docker image using azure.
+--------------------------------------------------------
+Prerequisite is you have an azure account and you have installed the azure cli, follow these steps to deploy the production Todo App docker image to docker hub and to run the web app using azure.
+1.	From a terminal window login to azure: ‘az login’
+2.	Build the docker file: ‘docker build --target production --tag helencurtis02/todo-app-prod:latest .’
+3.	Push the image to docker hub: ‘docker push helencurtis02/todo-app-prod:latest’
+4.	Image should now be in docker hub.
+5.	Next you need to create a web app within your azure portal environment. Create a WebApp resource for a docker container name ‘helcur-todoapp-prod’. 
+6.	For Docker image and tag use ‘helencurtis02/todo-app-prod:latest’.
+7.	Remaining settings should be defaults, add tags if you wish then click create.
+8.	From the azure portal for your web app, you will need to go to configuration and enter the following application settings adding values for your Trello board, do not forget to click Save once they have all been created.
+WEBSITES_PORT
+8000
+-----------------------------
+Name: FLASK_APP
+Value: todo_app/app
+-----------------------------
+Name: FLASK_ENV
+Value: production
+----------------------------
+Name: SECRET_KEY
+Value: [Your SECRET_KEY]
+----------------------------
+Name: TRELLO_BOARD_URL
+Value: [Your TRELLO_BOARD_URL]
+----------------------------
+Name: TRELLO_KEY
+Value: [Your TRELLO_KEY]
+----------------------------
+Name: TRELLO_TOKEN
+Value: [Your TRELLO_TOKEN]
+----------------------------
+Name: TRELLO_TODO_LIST
+Value: [Your TRELLO_TODO_LIST]
+----------------------------
+Name: TRELLO_DONE_LIST
+Value: [Your TRELLO_DONE_LIST]
+
+9.	Now browse to this url ‘http://helcur-todoapp-prod.azurewebsites.net/’and the website should be displayed in your browser.
```

Comment:
Tiny, but always try to use the HTTPS (secured) protocol over the HTTP one where possible 😄 

```suggestion
9.	Now browse to this url ‘https://helcur-todoapp-prod.azurewebsites.net/’and the website should be displayed in your browser.
```

Reply:
Updated